lib/_stream_duplexpair.js

```diff
@@ -1,15 +1,12 @@
 'use strict';
```

Comment:
Can you undo the file rename?

Reply:
@addaleax You mean you like `_stream_pair.js` better? I figured this is more consistent with `lib/internal/streams/duplexpair.js` and with the name of the property exported by "stream"

Reply:
Oh, so keep the stream under lib/internal, that makes sense